qiskit_nature/second_q/mappers/parity_mapper.py

```diff
@@ -40,6 +40,12 @@ class ParityMapper(FermionicMapper):
     (middle and last qubit) because the spin orbitals are ordered in two spin sectors
     (block spin order). Based on the provided number of particles this allows the automatic selection
     of the correct symmetry sector.
+
+    .. warning::
+
+       This additional two-qubit reduction will **not** work in conjunction with the
```

Comment:
Should we qualify `will not work` here. It will reduce it by two qubits, won't it. just that the resulting operator is likely not going to be "correct" - so using it the answer is likely to be wrong and hence will not work is more in that sense. It seemingly seems to work in that the code still runs, just that the computation it does is not likely to result in a correct answer.